ddb-metrics-collection/README.md

```diff
@@ -0,0 +1,164 @@
+# DynamoDB Metrics Collector
+
+## 📊 Uncover Hidden Insights in Your DynamoDB Tables
+
+Are your DynamoDB tables optimized for performance and cost? The DynamoDB Metrics Collector is here to help you find out! This powerful tool scans your AWS regions, identifies provisioned DynamoDB tables, and provides detailed utilization metrics to help you optimize your database infrastructure.
+
+![DynamoDB Metrics Collector Demo](./documentation/metrics.gif)
+
+## 🚀 Features
+
+- 🌎 **Multi-Region Support**: Scans all your AWS regions automatically
+- 🔍 **Smart Table Detection**: Identifies provisioned DynamoDB tables
+- 📈 **Comprehensive Metrics**: Collects read and write utilization data
+- 💡 **Utilization Insights**: Highlights tables with low utilization (below 45%)
+- 📊 **CSV Exports**: Generates easy-to-analyze CSV reports
+
+## 🛠 Installation
+
+This project is built with Poetry for dependency management. To get started:
+
+1. Clone the repository:
+
+`git clone https://github.com/awslabs/amazon-dynamodb-tools.git cd ddb-metrics-collection`
+
+
+2. Install dependencies with Poetry:
+poetry install
+
+
+Alternatively, you can use pip with the provided `requirements.txt`:
+pip install -r requirements.txt
+
+
+## 🏃‍♂️ Usage
+
+Run the metrics collection with a single command:
+
+```bash
+poetry run python ddb_metrics_collection/utilization_example.py --start-time 2025-02-19
```

Comment:
I do not plan on using poetry to run it. It's straighforward enough to remove the poetry run preamble, so I don't think this is a big deal.